test/integration/admissionconfig_test.go

```diff
@@ -294,6 +294,8 @@ func TestOpenshiftAdmissionPluginEmbeddedConfig(t *testing.T) {
 
 func TestAlwaysPullImagesOn(t *testing.T) {
 	testutil.RequireEtcd(t)
+	defer testutil.DumpEtcdOnFailure(t)
```

Comment:
Wouldn't you also want to do this with the test functions that call `setupAdmissionTest`?